### DISCUSSION TITLE
RaspberryMatic ausschließlich mit LAN-GWs nutzbar?

### DISCUSSION THREAD

**Discussion by elronzo** [OpenCCU/OpenCCU#2966] [🇩🇪 Fragen&Antworten]
Lässt sich RaspberryMatic ausschließlich mit HM-LAN-GWs betreiben? 
_(derzeit nutze ich es so unter FHEM, möchte aber zu RaspberryMatic wechseln)_

**Comment by jens-maus** [comment]
Wenn du nur ausschließlich BidCos-RF basierte Homematic-Geräte und keinerlei homematicIP (RF oder Wired) nutzt, dann ja.

**Comment by elronzo** [comment]
Ja. Habe nur Homematic Classic (BidCos-RF) Geräte im Einsatz. So könnte ich dann praktisch ein LAN-GW von FHEM abzwacken und gleich mal mit RaspberryMatic testen...

**Comment by elronzo** [comment]
Klareweise kann ich mir dann den Schritt "Prepare host to use Homematic and Homematic IP devices" laut [https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Kubernetes](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Kubernetes) dann auch sparen, wenn RaspberryMatic in einem K8s Pod läuft und die LAN-GWs auf abgesetzter Hardware... oder?

**Comment by elronzo** [comment]
Ich bin schon mal begeistert. Läuft in meiner K3s-Umgebung mit einem initialen Konfigurationsaufwand von unter 1 Minute.

In den Logs sehe ich noch "Starting Third-Party Addons: sh: write error: Permission denied". Ich nehme mal an, dass diese AddOns im Filesystem defaultmäßig woanders liegen als in /usr/local. Hier muss ich noch recherchieren...

Meine LAN-GWs sind derzeit unter Port 2000 erreichbar. In RaspberryMatic sehe ich aber keine Möglichkeit einen Port zu spezifizieren? Welcher Port wird hier erwartet? (Zusatzfrage: Kann ich anstelle der IP für einen LAN-GW auch einen FQDN angeben?) Aber ich fürchte fast, dass RaspberryMatic mit meiner derzeitigen, alten Hardware (HMUARTLGW) nicht kann... :-( 

Werde wohl also ein paar RPI-RF-MODs brauchen. Eines müsste sogar noch wo rumliegen...

**Reply by Baxxy13** [reply]
Für den reinen LGW-Betrieb reichen auch die alten HM-MOD-RPI-PCB. Es geht aber auch mit dem HmIP-RFUSB.

**Reply by elronzo** [reply]
> Für den reinen LGW-Betrieb reichen auch die alten HM-MOD-RPI-PCB. Es geht aber auch mit dem HmIP-RFUSB.

Gut zu wissen. Versuche ich in RaspberryMatic ein LGW zu konfigurieren, scheint dennoch etwas zu fehlen:

> No HomeMatic radio module or LAN gateway was found to be usable for homematic or homematicIP communication in your system.Please make sure that a compatible rf module or LAN gateway is correctly connected or configured so that you are able to teach-in devices.

Aus dem RaspberryMatic-Pod kann ich das LAN-Gateway aber tadellos (auf Port 2000) erreichen. 

Im Pod-Log sehe ich allerdings zwei Sachen, die mir nicht gefallen:
Starting rfd: ....................ERROR
Starting Third-Party Addons: sh: write error: Permission denied

**Reply by elronzo** [reply]
> Für den reinen LGW-Betrieb reichen auch die alten HM-MOD-RPI-PCB. Es geht aber auch mit dem HmIP-RFUSB.

Wie wäre das zu konfigurieren? Bekam es noch nicht funktionierend hin.

**Reply by Baxxy13** [reply]
Steht im Wiki:
https://github.com/jens-maus/RaspberryMatic/wiki/Experten-Features#lan-gateway-betrieb

**Reply by elronzo** [reply]
> Steht im Wiki: https://github.com/jens-maus/RaspberryMatic/wiki/Experten-Features#lan-gateway-betrieb

Danke. Möglicherweise ist meine Firmware (1.41) zu alt. Aber woher bekomm ich eine aktuellere? Auf https://github.com/eq-3/occu/tree/master/firmware/HM-MOD-UART wohl mal nicht. Und im Wiki steht  "[...] die Funktionalität RaspberryMatic als reinen LAN-Gateway zu nutzen erst ab Firmware 3.67.10.20230114 nicht nur mit einem HM-MOD-RPI-PCB [...]". Daraus würde ich eine Firmware-Version aus dem Jahre 2023 ableiten... aber wo finde ich die?

**Reply by MichaelN0815** [reply]
Du verwechselt FW von RaspberryMatic und dem Funkmodul. 3.67 ist eine RaspberryMatic Version. Und beim booten wird ggf das Funkmodul upgedatet.

**Reply by elronzo** [reply]
> Du verwechselt FW von RaspberryMatic und dem Funkmodul. 3.67 ist eine RaspberryMatic Version. Und beim booten wird ggf das Funkmodul upgedatet.

Danke! Auch gerade selbst herausgefunden.

Ich habe in Raspberrymatic bereits die IP für ein LAN-GW eingestellt. Funkioniert aber nix. Sieht so aus als würde es nicht gefunden.
